e2e_playwright/hello_app.py

```diff
@@ -14,10 +14,12 @@
 
 import numpy as np
 
+import streamlit as st
 from streamlit import runtime
 from streamlit.hello import streamlit_app
 
 # Set random seed to always get the same results in the plotting demo
 np.random.seed(0)
 if runtime.exists():
     streamlit_app.run()
+    st.logo("./static/streamlit-logo.png", size="large")
```

Comment:
nit: I think it's better to resolve the static path via a path relative to `__file__`, e.g.:

https://github.com/streamlit/streamlit/blob/a19d2d412c2ac8216017c4a91af9ae7c741623c1/e2e_playwright/st_html.py#L21